com.unity.ml-agents/Editor/AgentEditor.cs

```diff
@@ -22,6 +22,9 @@ public override void OnInspectorGUI()
                 new GUIContent("Max Step", "The per-agent maximum number of steps.")
             );
 
+            var observableAttributeBehavior = serializedAgent.FindProperty("m_observableAttributeHandling");
```

Comment:
I think the `m_observableAttributeHandling` belongs on the behavior parameters, not on the Agent inspector.

Reply:
Good call, I like that better.